lib/response.js

```diff
@@ -113,10 +114,19 @@ res.send = function send(body) {
   var encoding;
   var req = this.req;
   var type;
-
   // settings
   var app = this.app;
 
+  var isChunkBlob = !!(Blob && chunk instanceof Blob);
+
+  if (chunk && chunk.constructor && chunk.constructor.name === 'Blob') {
```

Comment:
Shouldn't this also be `if (isChunkBlob)`?

Reply:
I just removed this part